paper-drawer-panel.html

```diff
@@ -126,6 +134,8 @@
     }
 
     iron-selector > #main {
+      @apply(--paper-drawer-panel-main-container);
```

Comment:
I wasn't sure because the user will have to be very careful if we allow them to override the properties that make functional each element. For example:  ``position: absolute``.  I changed though.